ROS2 Humble, template build fail

<!-- If you're not sure whether your problem is a bug, please ask a question at
http://answers.gazebosim.org instead.-->

## Environment
* OS Version: Ubuntu 22.04
* Source or binary build? Template build

## Description
* Expected behavior: 
Building the template as per instruction
* Actual behavior: 
Got a build error, as in issue https://github.com/gazebosim/ros_gz/issues/402

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. Follow recipe in https://github.com/gazebosim/ros_gz_project_template#install

## Output
<!-- Provide screenshots, console logs, backtraces, and/or anything that could
be useful to us resolving this issue -->
```
colcon build --cmake-args -DBUILD_TESTING=ON
[0.334s] WARNING:colcon.colcon_core.package_selection:Some selected packages are already built in one or more underlay workspaces:
    'ros_gz_bridge' is in: /opt/ros/humble
If a package in a merged underlay workspace is overridden and it installs headers, then all packages in the overlay must sort their include directories by workspace order. Failure to do so may result in build failures or undefined behavior at run time.
If the overridden package is used by another package in any underlay, then the overriding package in the overlay must be API and ABI compatible or undefined behavior at run time may occur.

If you understand the risks and want to override a package anyways, add the following to the command line:
    --allow-overriding ros_gz_bridge

This may be promoted to an error in a future release of colcon-override-check.
Starting >>> ros_gz_interfaces
Starting >>> ros_gz_sim
Starting >>> sdformat_test_files
Starting >>> ros_gz_example_application
Starting >>> ros_gz_example_bringup
Starting >>> ros_gz_example_description
Starting >>> ros_gz_example_gazebo
Finished <<< sdformat_test_files [0.14s]                                                                  
Starting >>> sdformat_urdf
Finished <<< ros_gz_example_description [0.14s]
Finished <<< ros_gz_example_application [0.16s]
Finished <<< ros_gz_example_bringup [0.16s]
Finished <<< ros_gz_example_gazebo [0.19s]                                                              
Finished <<< ros_gz_sim [0.21s]
Finished <<< sdformat_urdf [0.17s]                                                                     
Finished <<< ros_gz_interfaces [1.08s]                      
Starting >>> ros_gz_bridge
--- stderr: ros_gz_bridge                              
Traceback (most recent call last):
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/bin/ros_gz_bridge_generate_factories", line 25, in <module>
    from ros_gz_bridge import generate_cpp
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/ros_gz_bridge/__init__.py", line 21, in <module>
    from rosidl_pycommon import expand_template
ModuleNotFoundError: No module named 'rosidl_pycommon'
gmake[2]: *** [CMakeFiles/ros_gz_bridge_lib.dir/build.make:80: generated/get_facUSERNAMEy.cpp] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:182: CMakeFiles/ros_gz_bridge_lib.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/bin/ros_gz_bridge_generate_tests", line 25, in <module>
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/bin/ros_gz_bridge_generate_tests", line 25, in <module>
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/bin/ros_gz_bridge_generate_tests", line 25, in <module>
    from ros_gz_bridge import generate_test_cpp
    from ros_gz_bridge import generate_test_cpp
    from ros_gz_bridge import generate_test_cpp
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/ros_gz_bridge/__init__.py", line 21, in <module>
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/ros_gz_bridge/__init__.py", line 21, in <module>
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/ros_gz_bridge/__init__.py", line 21, in <module>
    from rosidl_pycommon import expand_template
    from rosidl_pycommon import expand_template
    from rosidl_pycommon import expand_template
ModuleNotFoundError: No module named 'rosidl_pycommon'
ModuleNotFoundError: No module named 'rosidl_pycommon'
ModuleNotFoundError: No module named 'rosidl_pycommon'
Traceback (most recent call last):
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/bin/ros_gz_bridge_generate_tests", line 25, in <module>
    from ros_gz_bridge import generate_test_cpp
  File "/home/USERNAME/template_ws/src/ros_gz/ros_gz_bridge/ros_gz_bridge/__init__.py", line 21, in <module>
    from rosidl_pycommon import expand_template
ModuleNotFoundError: No module named 'rosidl_pycommon'
gmake[2]: *** [CMakeFiles/test_ros_publisher ...
```

--- a/template_workspace.yaml
+++ b/template_workspace.yaml
@@ -8,7 +8,7 @@ repositories:
   ros_gz:
     type: git
     url: https://github.com/gazebosim/ros_gz.git
-    version: ros2
+    version: humble
   sdformat_urdf:
     type: git
     url: https://github.com/ros/sdformat_urdf.git

